rrfs-test/testinput_expr/rrfs_fv3jedi_hyb_2022052619.yaml

```diff
@@ -257,16 +210,19 @@ variational:
       npz: 60
       layout: [8,10]
       io_layout: [1,1]
-      ntiles: 1
+      ntiles: 1 
       fieldsets:
       - fieldset: DataFix/fix/dynamics_lam_cmaq.yaml
+    diagnostics:
+      departures: ombg
 final:
   diagnostics:
     departures: oman
 
 output:
   filetype: fms restart
   datapath: ./
-  prefix: hybens3dvar-fv3_lam-C775
+  prefix: hybens3dvar-fv3_lam-C775 #Ens3dvar-fv3_lam-C775
```

Comment:
Though the yaml is with "hyb" in its name, it is actually a pure ens3dvar run,  not sure why the name is "hybens...".  But that is ok for being now. In the future, we could make it a real hyb run if we don't keep a pure ens3dvar ctest.

Reply:
Is this yaml not running the hybrid? There is a setting for `covariance model: hybrid` and then there are two covariance sections (`ensemble` and `SABER`).

Reply:
@SamuelDegelia-NOAA  Sorry, I missed that "hybrid " session with the nicas part.  Ok then, that is a hybrid run, while the static B is of unit variances.